src/components/views/rooms/LinkPreviewWidget.js

```diff
@@ -128,15 +128,15 @@ module.exports = createReactClass({
         }
 
         const AccessibleButton = sdk.getComponent('elements.AccessibleButton');
+        // Escape </> to prevent any HTML injections, we can't replace & as the description may contain & encoded html entities
+        const safeDescription = (p["og:description"] || "").replace("<", "&lt;").replace(">", "&gt;");
```

Comment:
```
">>>><<<<".replace("<", "&lt;").replace(">", "&gt;")
"&gt;>>>&lt;<<<"
```

We should leave this problem for one of our many sanitization libraries to solve.

Reply:
Are you suggesting sanitize-html here? That would have a different function, it'd turn `<a>foobar</a>` into `foobar` instead of `&lt;a&gt;foobar&lt;/a&gt;`

Reply:
I thought we had one but it turns out we didn't. I ended up solving this with https://github.com/matrix-org/matrix-react-sdk/pull/3802/files#diff-810e1fe41fce260c854536367753bc2bR532-R535 in a different PR.

Reply:
going to duplicate that method, thanks

Reply:
Actually, it won't work here as it'll escape the `&` to `&amp;`

Reply:
What's the issue with escaping &? It should be fine?

Reply:
oh, I see, partially escaped HTML isn't really our problem...

Reply:
escaping `&` means it'll end up doing nothing as we escape the html-escape sequences we actually desire to render

Reply:
This might call for https://www.npmjs.com/package/html-entities